Makefile

```diff
@@ -102,15 +107,17 @@ tests-generate: tests-generate-state tests-generate-difficulty ## Generate all t
 tests-generate-state: ## Generate state tests.
 	@echo "Generating state tests."
 	env COREGETH_TESTS_GENERATE_STATE_TESTS=on \
+	env COREGETH_TESTS_CHAINCONFIG_FEATURE_EQUIVALENCE_COREGETH=on \
```

Comment:
`COREGETH_TESTS_CHAINCONFIG_FEATURE_EQUIVALENCE_COREGETH` forces all `Forks` values used by the tests to be `confp.Convert`ed on `init` to `&coregeth.ChainConfig` type values. These config values are dumped JSON encoded to an adjacent file. 

No longer do clients' test runner have to somehow magically know what "Berlin" means; they can look it up and see exactly what config should be used.